content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Virtual Machine Installation
+description: Deploy istio and connect a virtual machine to it.
+weight: 40
+keywords:
+- kubernetes
+- virtual-machine
+- gateways
+- vms
+---
+
+Follow this guide to deploy Istio and connect a virtual machine to it.
+
+{{< warning >}}
+This guide relies heavily on the
+[stable plugin ca certificate](/docs/tasks/security/cert-management/plugin-ca-cert/) feature. Istio
+does not offer an industry standard recommendation for certificate management. Please consult your
+information security team when using any plugin certificate. The determination of your company
+policies for certificate management is the choice of your company.
+{{< /warning >}}
+
+{{< tip >}}
+This guide is tested and validated. The Istio authors feel this guide is suitable for experimentation
+but not production. Like all alpha features, this guide is subject to change.
+{{< /tip >}}
+
+## Prerequisites
+
+1. [Download the Istio release](/docs/setup/getting-started/#download)
+1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/)
+1. Check the requirements [for Pods and Services](/docs/ops/deployment/requirements/)
+1. Virtual machines must have IP connectivity to the ingress gateway in the connecting mesh, and optionally every pod in the mesh via L3 networking if enhanced performance is desired.
+
+## Prepare the guide environment
+
+1. Set the environment variables `"${ISTIO_DIR}"`, `"${WORK_DIR}"`, your cluster
+    name, and the service namespace. Ensure `"${WORK_DIR}"` is prefixed with `"${WORK_DIR}"`
+    (e.g. `WORK_DIR="${WORK_DIR}"/vmintegration`).
```

Comment:
The `Ensure ...` part doesn't seem necessary, or even make sense. Why should I need to set WORK_DIR twice? I suggest deleting.
```suggestion
    name, and the service namespace.
```

Reply:
great catch Frank. Problem Addressed.

Cheers
-steve

Reply:
fwiw, the reason the recommendation is there, is all commands are based upon absolute paths.